documentation/docs.md

```diff
@@ -0,0 +1,222 @@
+# Authentication View
```

Comment:
I would prefer instead of having a separate docs file with these additions, that we instead add the same info into the class file itself, and we write it using qdoc format. For example, add the AuthenticationView documentation and code snippets in here - https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/main/uitools/import/Esri/ArcGISRuntime/Toolkit/AuthenticationView.qml

This way, the doc is all centralized, and we also end up with documentation showing up on our developer site - https://developers.arcgis.com/qt/toolkit/api-reference/qml-authenticationview.html